src/sections/evidence/EuropePmc/Body.js

```diff
@@ -59,29 +59,32 @@ const columns = [
 ];
 
 // Merges data from platform-API and EuropePMC API.
-const mergeData = (rows, literatureData) => {
-  const mergedRows = [...rows];
-
-  rows.forEach(row => {
+function mergeData(rows, literatureData) {
```

Comment:
This is a fix to a bug unrelated to the PR but it was blocking my work because it was throwing an error in dev mode.